.travis.yml

```diff
@@ -16,16 +16,11 @@ install:
 
 before_script:
   - make doc
-  - if [[ `git status --porcelain` ]]; then
-      echo "'make doc' was not run before commiting";
-      exit 1;
-    fi
+  - git status --porcelain || (echo "'make doc' was not run before commiting" && return 1)
```

Comment:
This is reversed and you need condition brackets
`[[ `git status --porcelain` ]] && (echo "'make doc' was not run before commiting" && return 1)`

Reply:
Also `exit` works globally while `return` only works in functions and sourced scripts. Are you sure it works here?

Reply:
The condition brackets are not required on this case. `git status --procelain` already returns 0 if the folder is not changed and non zero otherwise.

Reply:
The exit caused the output to be truncated.

Reply:
Locally, the only thing that works for me is ```
[[ `git status --porcelain` ]] && (echo "'make doc' was not run before commiting" && return 1)
```

Reply:
Strange, I will check that with you at the office.